doc/src/devdocs/callconv.md

```diff
@@ -16,7 +16,8 @@ signature.
 
   * LLVM ghosts (zero-length types) are omitted.
   * LLVM scalars and vectors are passed by value.
-  * LLVM aggregates (arrays and structs) are passed by reference.
+  * LLVM aggregates (arrays and structs) are [passed by
+    sharing](https://en.wikipedia.org/wiki/Evaluation_strategy#Call_by_sharing) (i.e. as pointers).
```

Comment:
This refers to the ABI, so I don't think the call-by-sharing terminology is really necessary or helpful.